pkg/distributor/distributor.go

```diff
@@ -335,6 +356,15 @@ func New(
 		writeFailuresManager: writefailures.NewManager(logger, registerer, cfg.WriteFailuresLogging, configs, "distributor"),
 		kafkaWriter:          kafkaWriter,
 		partitionRing:        partitionRing,
+		limitsFrontendRing:   limitsFrontendRing,
+		limitsFrontends: limits_frontend_client.NewPool(
+			"ingest-limits-frontend",
```

Comment:
nit. I suggest to use the constant for the ring name in the frontend package.

Reply:
Oh nice one! We do have a constant for this, and I forgot to use it here.